frontend/src/component/integrations/IntegrationList/AvailableIntegrations/SDKs.ts

```diff
@@ -58,8 +58,8 @@ export const OFFICIAL_SDKS: Sdk[] = [
     },
     {
         name: 'dotnet',
-        displayName: '.Net SDK',
-        description: 'Officially Unleash Client for .Net',
+        displayName: '.NET SDK',
+        description: 'Officially Unleash Client for .NET',
```

Comment:
```suggestion
        description: 'Official Unleash Client for .NET',
```

I know you fixed some of these in the previous PR, but there are still more.  All the `officially` in this file can be replaced with `official`.